database/src/pool/postgres.rs

```diff
@@ -631,6 +637,7 @@ where
                             Profile::from_str(row.get::<_, String>(2).as_str()).unwrap(),
                             row.get::<_, String>(3).as_str().parse().unwrap(),
                             CodegenBackend::from_str(row.get::<_, String>(4).as_str()).unwrap(),
+                            Target::from_str(row.get::<_, String>(6).as_str()).unwrap(),
```

Comment:
```suggestion
                            Target::from_str(row.get::<_, String>(5).as_str()).unwrap(),
```
And the line below need to use `(6)`.